web/js/precallController.js

```diff
@@ -90,14 +108,16 @@
         }
 
         otHelper.initPublisher('video-preview',
-          { width: '100%', height: '100%', insertMode: 'append', showControls: false }
+          Object.assign(
+            { width: '100%', height: '100%', insertMode: 'append', showControls: false }, publisherOptions)
```

Comment:
Instead of using `Object.assign()` to add these fixed properties to `publisherOptions`, can you simply define these properties when you instantiate `publisherOptions`:

```
  var publisherOptions = {
    publishAudio: true,
    publishVideo: true,
    name: '',
    width: '100%',
    height: '100%',
    insertMode: 'append',
    showControls: false
  };
```